foodalert/urls.py

```diff
@@ -20,4 +20,8 @@
     url(r'^updates/$', UpdateList.as_view(), name='update_list'),
     url(r'^updates/(?P<pk>[0-9]+)/$',
         UpdateDetail.as_view(), name='update_detail'),
+    url(r'^subscription/$', SubscriptionList.as_view(),
+        name='subscription_list'),
+    url(r'^subscription/(?P<pk>[0-9]+)/$',
+        SubscriptionDetail.as_view(), name='update_detail'),
```

Comment:
name='update_detail' should be name='subscription_detail' in the URL for the subscription detail endpoint